src/main/java/com/microsoft/aad/msal4j/Event.java

```diff
@@ -68,10 +69,17 @@ static String scrubTenant(URI uri){
             return null;
         }
 
-        //TODO should be updated when B2C is added, since tenant could be in different place
         String[] segment = uri.getPath().split("/");
         if(segment.length >= 2){
-            segment[1] = TENANT_PLACEHOLDER;
+            if(segment[1].equals("tfp")){
+                segment[2] = TENANT_PLACEHOLDER;
```

Comment:
>segment[2] = TENANT_PLACEHOLDER; [](start = 16, length = 32)

if length is 2 wouldn't this result in an error?

Reply:
Yes, but we are implicitly checking that segment[2] will not be null by checking that segment[1] == "tfp" (if segment[1] == "tfp" then segment["2"] will never be null). This is a good point though, it is probably better to have the explicit check anyways. I will update.